docs/modules/ROOT/pages/vshn-managed/postgresql/sla.adoc

```diff
@@ -0,0 +1,32 @@
+= Service Level Agreement
+
+[IMPORTANT]
+====
+The Service Level Agreement can be chosen during a PostgreSQL instance request.
+It can be changed later on and the price will be updated accordingly.
```

Comment:
```suggestion
It can be changed later on. The price together with the provided service will be updated accordingly. The change takes effect on the 1st of the next month.
```